_data/de/site.yml

```diff
@@ -29,8 +29,8 @@ footer:
   archives: Archiv
   process: Prozess
   coc: Code of Conduct
-  feedback: Wir suchen nach Feedback
-  feedback-prompt: Bitte öffnen Sie <a href="https://github.com/tc39/tc39.github.io/issues/new" target="_blank">ein Issue </a> in <a href="https://github.com/tc39/tc39.github.io" target="_blank">unser Website-Repository</a>.
+  feedback: Wir sind auf der Suche nach Feedback.
```

Comment:
There are certainly alternatives. I tried to keep it close to the original English phrasing.

Reply:
I think, we can always use what is more common in the locale's language if it is more natural. But whatever you choose would work for me!